### ISSUE TITLE
Initial Batch Normalization

### ISSUE THREAD

**Issue by MustaphaBM** [dreamquark-ai/tabnet#305]
In TabNet Encoder there is a initial BatchNorm1D Layer also in TabNetNoEmbedding there is an initial BatchNorm1D Layer
In the paper they mentioned that only the input features will pass through BatchNorm1D Layer. So the initial_bn must be only in TabNet Encoder.

There is two initial_batch normalization
**What is the current behavior?**
input data will pass through
